experiments/experiments.go

```diff
@@ -10,6 +10,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gofrs/uuid"
```

Comment:
nit: Also please add a blank line after this line so it's in a different group with baseplate.go packages.